docs/content/en/use-cases/wildcarded.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Pass user names and passwords as is
+menuTitle: Wildcarded users
+category: Use Cases
+position: 305
+---
+
+Suppose you need to use `ClickHouse` LDAP facilities. It is more secure and more flexible than having credentials hardcoded in ClickHouse or a `chproxy` configuration files.
+The following `chproxy` config may be used for [this use case](https://github.com/ContentSquare/chproxy/blob/master/config/examples/wildcarded.yml):
+```yml
+log_debug: true
+
+users:
+  # wildcarded user
+  # matches with any name with prefix 'analyst_'
+  # e.g. 'analyst_joe' or 'analyst_jane'
+  - name: "analyst_*"
+    to_cluster: "default"
+    to_user: "analyst_*"
```

Comment:
in order to add  extra safety (because people could you crazy login involving "_*" even if it's very unlikely) and to make the feature not rely on naming convention, you should add an optional boolean field is_wildcarded (false by default) to activate or not the feature for the user.
Doing so would also make the function  getUser() a bit faster in the worst cases (many users and a few wildcarded) since you could skip useless string manipulation operations.

Reply:
Hello, it's a valid point, but are you sure that we need an explicit flag? From performance standpoint it is enough to implicitely switch this feature if a wildcarded user is actually present in config.
If the answer is "yes, we do need an explicit flag", I think we should raise an error if the flag is not set and a wildcarded user is found in config, right?